docs/faq.md

```diff
@@ -0,0 +1,14 @@
+# Frequrently Asked Questions: [WIP]
+
+* [What is an Ingress Controller?](#what-is-an-ingress-controller?)
+* [Can single ingress controller instance manage multiple Application Gateway?](#can-single-ingress-controller-instance-manage-multiple-application-gateway?)
+
+## What is an Ingress Controller?
+Kubernetes allows creation of `deployment` and `service` resource to expose a group of pods internally in the cluster. To expose the same service externally, an [`Ingress`](https://kubernetes.io/docs/concepts/services-networking/ingress/) resource is defined which provides load balancing, SSL termination and name-based virtual hosting.
```

Comment:
Should we also mention here that the `Service` definition is not much more than context for AGIC - I remember some of our customers were confused by the IP address that was assigned to the Service (I believe when of type: LoadBalancer)